composer.json

```diff
@@ -22,9 +22,9 @@
     "require": {
       "php": "^5.6.0 || ^7.0.0",
       "psr/log": "^1.0.0",
-      "symfony/console": "~2.0 || ~3.0",
-      "symfony/event-dispatcher": "~2.0 || ~3.0",
-      "symfony/monolog-bridge": "~2.0 || ~3.0"
+      "symfony/console": "^2.0 || ^3.0",
+      "symfony/event-dispatcher": "^2.0 || ^3.0",
+      "symfony/monolog-bridge": "^2.0 || ^3.0"
```

Comment:
but why not `^4.0`?

Reply:
because I haven't tested on 4. My project is only 3. There seem to be major changes on Symfony 4. So unless I have checked it myself, I wont add it in to break the console classes.